scinim/numpyarrays.nim

```diff
@@ -0,0 +1,182 @@
+import std/sequtils
+import std/strformat
+import std/tables
+
+import arraymancer
+
+import nimpy
+from nimpy {.all.} import newPyObject, nimValueToPy
```

Comment:
It"s a possibility, but I'm not sure those are meant to be public API. 

We just need it on this specific use case because do low-level stuff and using the private import syntax was much easier than doing a work around based on ``include``